raft.c

```diff
@@ -199,6 +199,40 @@ static void executeLogEntry(RedisRaftCtx *rr, raft_entry_t *entry, raft_index_t
     }
 }
 
+static void raftSendNodeRemoved(raft_node_t *raft_node)
```

Comment:
I'd call it raftSendNodeShutdown (and throughout).  Yes, its saying "you've been removed", but its really sending a shutdown command.

Reply:
I see. I thought leader sends "node removed" message, what to do is up to the receiver, so we may change the behavior, decide not to shutdown and do something else without changing the protocol (old habit from protocol jobs). But I see no need this here. I'll change it to "node shutdown".  Thanks.